public/index.html

```diff
@@ -57,7 +57,9 @@
             
             <!-- Leave prod version commented for development -->
             <!-- Prod version -->
-            <li><a href="/faqs" class="anchor">FAQs</a></li>
+            <li><a href="https://intercom.help/poap/en/" class="anchor">FAQs</a></li>
+
+            <li><a href="https://poap.notion.site/Working-POAP-53f427aac2df4bca8cb6292feabfec47" class="anchor">Careers</a></li>
```

Comment:
Please add target="_blank" on external links so they open on a new tab

Reply:
Added to all external links (not just these).